### DISCUSSION TITLE
Wann ist mit einem neuen Release des Nugets zu rechnen?

### DISCUSSION THREAD

**Discussion by Heinz164** [stephanstapel/ZUGFeRD-csharp#414] [General]
Zuletzt wurde ein Fehler behoben, durch den der Identifier (und somit die E-Rechnung) nicht erkannt wurde, sofern im Identifier Großbuchstaben vorgekommen sind. Deshalb würde mich interessieren, bis wann mit einem neuen Release des Nugets zu rechnen ist, in dem dieser Fehler behoben ist.

**Comment by stephanstapel** [comment]
Im Prinzip jederzeit. Ich warte nur noch auf Feedback zu https://github.com/stephanstapel/ZUGFeRD-csharp/issues/408 und https://github.com/stephanstapel/ZUGFeRD-csharp/issues/398

**Comment by stephanstapel** [comment]
@Heinz164 : hier ist es:
https://www.nuget.org/packages/ZUGFeRD-csharp/
